dll/cpl/timedate/internettime.c

```diff
@@ -379,6 +446,12 @@ InetTimePageProc(HWND hwndDlg,
             {
                 case PSN_APPLY:
                     SetNTPServer(hwndDlg);
+
+                    if (SendDlgItemMessage(hwndDlg, IDC_AUTOSYNC, BM_GETCHECK,0,0) == BST_CHECKED)
```

Comment:
Missing some spaces after the commas here.

Reply:
Yes, I will add spaces after the commas. Thanks.